lib/index.js

```diff
@@ -162,6 +165,76 @@ export default {
     return provider
   },
 
+  provideHyperclick(): HyperclickProvider {
```

Comment:
Oops, I was missing the declaration. Will add it.

Why not add the plugin since it's just for the linter?